packages/@okta/vuepress-site/docs/reference/password-hook/index.md

```diff
@@ -27,7 +27,7 @@ The Password Import Inline Hook enables migration of users from another data sto
 
 The Password Import Inline Hook is triggered when the end user tries to sign in to Okta for the first time. Okta sends your external service the password that the user supplied. Your external service then needs to send a response to Okta indicating whether the password supplied by the end user is valid or not.
 
-If the password is valid, Okta can authenticate the user independently from then on.
+When your service returns a response that indicates that the password is valid, Okta sets the password for the user and won't call your service again. However, if Okta service is temporarily disrupted or degraded, Okta might not be able to set the password, and will need to call your service on subsequent authentication attempts until the password is successfully set. See [Password Inline Hook and Okta Service Mode](#password-inline-hook-and-okta-service-mode) and [Removing Password from Existing User Store](#removing-password-from-existing-user-store) for details.
```

Comment:
nit : not sure if we should use the word `set` password or `import` password. Same way, in the end - until the password is successfully `set` vs. until the password is successfully `imported`.